### PR TITLE
fix: unknown tensorboard image format when logging gifs.

### PR DESCRIPTION
**Pull request by jacobromero**
Description
-----------

- Fixes WB-20916

What does the PR do? Include a concise description of the PR contents.

This PR fixes an unknow image format error that occurs when trying to upload a GIF via tensorboard sync.

- [x] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?
- system tests
- unit tests
- manual testing
  - Attempted to upload a gif via SummaryWriter `write_video` before changes -> the run failed
  - Made changes and attempted to upload a gif via SummaryWriter `write_video` -> the run passed
  - Validated above against legacy behavior

### CODE REVIEW

File: core/internal/tensorboard/tfeventconverter_test.go [hunk 1]
```diff
@@ -27,6 +27,14 @@ const testPNG2x4 = "" +
 	"\x01\x00\x00\x00\x00" + // buncha other stuff
 	"\x8C\x94\xD3\x94" // CRC-32 of "IHDR" and the chunk data
 
+const testGif1x1 = "" +
+	// GIF header
+	"GIF89a" +
+	// Gif size (1x1)
+	"\x01\x00\x01\x00" +
+	// random Gif data
```
Comment (kptkin) [inline core/internal/tensorboard/tfeventconverter_test.go:35]: random zeros ;)

File: tests/system_tests/test_functional/test_tensorboard/test_tensorboardX.py
Comment (kptkin) [inline tests/system_tests/test_functional/test_tensorboard/test_tensorboardX.py:1]: why did you choose to use tensorboardX? maybe better to use tensorboard or the torch.tensorboard

**Comment by kptkin** [comment]
I think this one is good to mention in the changelog, just in case some of our users noticed that we didn't have support for it before

**Review by kptkin — APPROVED** [review]
Nicely done!

File: tests/system_tests/test_functional/test_tensorboard/test_tensorboardX.py
Comment (jacobromero) [reply tests/system_tests/test_functional/test_tensorboard/test_tensorboardX.py:1]: No particular reason for tensorboardX, in fact it should be the same test for torch tensorboard as well.
Let me add a test there for completeness.

Comment (kptkin) [reply tests/system_tests/test_functional/test_tensorboard/test_tensorboardX.py:1]: yeah it is just the least maintained package: https://github.com/lanpa/tensorboardX stopped updating it in 2022

File: core/internal/tensorboard/tfeventconverter_test.go (hunk 1 above)
Comment (jacobromero) [reply core/internal/tensorboard/tfeventconverter_test.go:35]: https://xkcd.com/221/
😃

**Pull request merged**